<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="ticketsLabel">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Tickets</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="selectTicketLabel">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Select ticket first!</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="ticketList"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="createTicketButton">
                  <property name="text">
                   <string>Create</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="infoTicketButton">
                  <property name="text">
                   <string>Info</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="editTicketButton">
                  <property name="text">
                   <string>Edit</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="employeesLabel">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Employees</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="selectEmpLabel">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Select employee first!</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="employeeList"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="addEmployeeButton">
                  <property name="text">
                   <string>Add new</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="saveScheduleButton">
                  <property name="text">
                   <string>Save schedule</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="removeEmployeeButton">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="3" column="0">
               <widget class="QLabel" name="legendLabel">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>White - free time slots,  Blue - used time slots</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QTableWidget" name="calendar">
                <property name="maximumSize">
                 <size>
                  <width>780</width>
                  <height>300</height>
                 </size>
                </property>
                <row>
                 <property name="text">
                  <string>8:00-9:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>9:00-10:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>10:00-11:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>11:00-12:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>12:00-13:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>13:00-14:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>14:00-15:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>15:00-16:00</string>
                 </property>
                </row>
                <row>
                 <property name="text">
                  <string>16:00-17:00</string>
                 </property>
                </row>
                <column>
                 <property name="text">
                  <string>Mon</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Tue</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Wed</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Thu</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Fri</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Sat</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Sun</string>
                 </property>
                </column>
                <item row="0" column="0">
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
                <item row="1" column="0">
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
                <item row="2" column="0">
                 <property name="flags">
                  <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
                 </property>
                </item>
                <item row="3" column="0"/>
                <item row="4" column="0"/>
                <item row="5" column="0"/>
                <item row="6" column="0"/>
                <item row="7" column="0"/>
                <item row="8" column="0"/>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="scheduleLabel">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Schedule: not specified</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
